src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons.tsx

```diff
@@ -354,15 +355,64 @@ export const ArtworkSidebarCommercialButtons: React.FC<ArtworkSidebarCommercialB
     artwork.isAcquireable || activePartnerOffer
   )
 
+  const SaleMessageOrOfferDisplay: FC = () => {
+    if (activePartnerOffer) {
```

Comment:
Judging from https://github.com/artsy/force/pull/13660/files#diff-7e67206208c67ae3f411657c84fa58ff49ccc4f44682f608a1ce8d56b46d4720R81-R82 I think `activePartnerOffer` will disappear when the timer runs out. In this case the offer from the gallery would disappear on a reload, but if the timer runs out while the user is on the page do we want the pricing to just suddenly change, timer disappear, etc? Not certain what the right answer is here.

If we don't want it to disappear completely, the variable we can use is just `partnerOffer.`

Reply:
Updated this part.